### PR TITLE
Feature/cdd 2363

### PR DESCRIPTION
**Pull request by luketowell**
# Description

These commits include the changes required in order to add a key to the weather health alerts map as well as the UKHSA logo. 

The implementation of the key matches the design attached to the ticket. I have also made the use of colour in the region modal and the key consistent with the map colours. 

Fixes #CDD-2636

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [ x ] Unit tests
- [ x ] Playwright e2e tests
- [ ] Mobile responsiveness
- [ ] Accessibility (i.e. Lighthouse audit)
- [ ] Disabled JavaScript

# Checklist:

- [ x ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ x ] Any styles in this change follow the 'STYLES.md' guide
- [ ] My changes are progressively enhanced with graceful degredagation for older browsers and non-JavaScript users
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ x ] I have added tests that prove my fix is effective or that my feature works
- [ x ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
   "typescript.tsdk": "node_modules/typescript/lib",
-  "typescript.enablePromptUseWorkspaceTsdk": true
+  "typescript.enablePromptUseWorkspaceTsdk": true,
+  "cSpell.words": ["Colour"]
```
Comment (luketowell) [inline .vscode/settings.json:4]: vs code was really winding me up by flagging 'Colour' as a spelling mistake.

File: src/app/components/ui/ukhsa/Map/shared/controls/HealthAlertControl.tsx [hunk 2]
```diff
@@ -73,8 +73,11 @@ const AlertDialogContent = () => {
             <SummaryListRow>
               <SummaryListKey>{t('map.alertDialog.statusKey')}</SummaryListKey>
               <SummaryListValue>
-                <div className={clsx(`govuk-tag capitalize`, getTagVariantFromStatus(status))}>
-                  {status.toLowerCase()}
+                <div
+                  className={clsx(`govuk-tag capitalize`, getTextColourCssFromColour(status))}
```
Comment (rhys-burendo) [inline src/app/components/ui/ukhsa/Map/shared/controls/HealthAlertControl.tsx:77]: We avoid using html style props on components, but wagtail should be able to handle this, potentially a classname of like

```suggestion
                  className={clsx(`govuk-tag capitalize`, getTextColourCssFromColour(status), `bg-${getCssVariableFromColour(status)} )}
```

Might do the trick - have a little experiment with these?

Comment (luketowell) [reply src/app/components/ui/ukhsa/Map/shared/controls/HealthAlertControl.tsx]: I have removed all instances where style was used and replaced with classes as is the standard

File: e2e/fixtures/pages/weather-health-alerts/weather-health-alerts-child.fixture.ts [hunk 3]
```diff
@@ -56,12 +56,13 @@ export class WeatherHealthAlertsChildPage {
...
 
     for (let i = 0; i < alertList.length; i++) {
+      const expectedStatus = alertList[i].status === 'Green' ? 'No alert' : `${alertList[i].status.toLowerCase()} alert`
       const listItem = regions.getByRole('listitem').nth(i)
       await expect(listItem).toBeVisible()
 
       await expect(listItem.getByRole('heading', { level: 2, name: alertList[i].region })).toBeVisible()
       await expect(listItem.getByText(alertList[i].updated)).toBeVisible()
-      await expect(listItem.getByText(alertList[i].status, { exact: true })).toBeVisible()
+      await expect(listItem.getByText(`${expectedStatus}`, { exact: true })).toBeVisible()
```
Comment (rhys-burendo) [inline e2e/fixtures/pages/weather-health-alerts/weather-health-alerts-child.fixture.ts:65]:
```suggestion
      await expect(listItem.getByText(expectedStatus, { exact: true })).toBeVisible()
```

**Review by rhys-burendo — APPROVED** [review]

**Review by rhys-burendo — APPROVED** [review]

**Pull request merged**